src/cms/templates/pois/poi_form.html

```diff
@@ -109,14 +109,18 @@ <h1 class="heading">
                         <span class="inline-block mb-2 font-bold">{% trans 'Status' %}:</span>
                         {{ poi_translation_form.instance.get_status_display }}
                     {% endif %}
-                    <label for="{{ poi_translation_form.title.id_for_label }}" class="block mb-2 mt-4 font-bold">{{ poi_translation_form.title.label }}</label>
+                    <label for="{{ poi_translation_form.title.id_for_label }}" 
+                    data-slugify-url="{% url 'slugify_ajax' region_slug=region.slug language_slug=language.slug %}{% if poi_form.instance.id %}?page={{ poi_form.instance.id }}{% endif %}"
+                    class="block mb-2 mt-4 font-bold">{{ poi_translation_form.title.label }}
+                    </label>
                     {% render_field poi_translation_form.title|add_error_class:"border-red-500" class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                   
                     <label for="{{ poi_translation_form.slug.id_for_label }}" class="block my-2 font-bold">{{ poi_translation_form.slug.label }}</label>
                     <div class="my-2 text-s text-gray-600">{{ poi_translation_form.slug.help_text }}</div>
                     <div id="slug-div" class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400">
                         {% spaceless %}
                             <div style="display: table; white-space: nowrap;">
-                                <span style="display: table-cell;">https://integreat.app/{{ region.slug }}/{{ language.slug }}/</span>
+                                <span style="display: table-cell;">https://integreat.app/{{ region.slug }}/{{ language.slug }}/{{ poi_translation_form.instance.slug }}</span>
```

Comment:
Please make this look similar to the Page form - in the moment, it still looks like before.
![Screenshot_2021-04-26 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/116117304-5d0b0d80-a6bc-11eb-97bc-83d8887e6bbd.png)